Assets/Scripts/Client/Player/PlayerBehavior.cs

```diff
@@ -13,25 +23,64 @@ public void Start()
 
   public void Update()
   {
+    movementTimePassed -= Time.deltaTime;
+    rotateTimePassed -= Time.deltaTime;
+
     if (!isLocalPlayer)
     {
       return;
     }
-    if (Input.GetKeyDown(KeyCode.W))
-    {
-      CmdRotate();
-    }
-    if (Input.GetKeyDown(KeyCode.S))
-    {
-      CmdMoveDown();
-    }
-    else if (Input.GetKeyDown(KeyCode.A))
+
+    if(movementTimePassed <= 0 || !movementKeyPressedLately)
     {
-      CmdMoveLeft();
+      movementTimePassed = movementKeyDelay;
+      movementKeyPressedLately = false;
+
+      if (Input.GetKey(KeyCode.S))

```

Comment:
Should we be letting you move Down and Left on the same frame? I feel like in normal tetris you can only do one or the other.